### PR TITLE
CLOUDP-70059: mongocli ops-manager|cloud-manager backups config list|ls

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70059

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli om backup config describe 5f719b352845cb05726a86f9 -P ops -o json
{
  "groupId": "5f6cc16511c3aa3a90b5ec34",
  "clusterId": "5f719b352845cb05726a86f9",
  "statusName": "STARTED",
  "storageEngineName": "WIRED_TIGER"
}
```

```
./bin/mongocli om backup config describe 5f719b352845cb05726a86f9 -P ops
PROJECT ID                 CLUSTER ID                 STATUS    STORAGE ENGINE
5f6cc16511c3aa3a90b5ec34   5f719b352845cb05726a86f9   STARTED   WIRED_TIGER
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
waiting https://github.com/mongodb/mongocli/pull/463

**Comment by andreaangiolillo** [comment]
@gssbzn tagging you here so that you can review this PR once you're back

File: internal/store/backup_configs.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		return result, err
+	default:
+		return nil, fmt.Errorf("unsupported service: %s", s.service)
+	}
+}
+
+// ListBackupConfigs encapsulates the logic to manage different cloud providers
+func (s *Store) ListBackupConfigs(projectID string, options *atlas.ListOptions) (*opsmngr.BackupConfigs, error) {
+	switch s.service {
+	case config.OpsManagerService:
```
Comment (gssbzn) [inline internal/store/backup_configs.go:51]: [nit] this should work for cloud manager as well, https://docs.cloudmanager.mongodb.com/reference/api/backup-configurations/

File: internal/store/backup_configs.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"fmt"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+
```
Comment (gssbzn) [inline internal/store/backup_configs.go:22]: [nit] extra space

File: internal/cli/opsmanager/backup/config/describe_test.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+// +build unit
+
+package config
+
+import (
+	"testing"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/flag"
+
```
Comment (gssbzn) [inline internal/cli/opsmanager/backup/config/describe_test.go:24]: extra space

File: internal/cli/opsmanager/backup/config/describe.go [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli ops-manager backup config describe clusterID [--projectId projectId]
+func DescribeBuilder() *cobra.Command {
+	opts := &DescribeOpts{}
+	cmd := &cobra.Command{
+		Use:     "describe [clusterID]",
```
Comment (gssbzn) [inline internal/cli/opsmanager/backup/config/describe.go:56]: [nit] left this comment in a separate PR but if the id is mandatory then this should be 
```suggestion
		Use:     "describe <clusterID>",
```

**Review by gssbzn — CHANGES_REQUESTED** [review]
happy to keep this instead of one for get one for list, couple of nits

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**